src/fidesops/api/v1/endpoints/dataset_endpoints.py

```diff
@@ -161,6 +195,33 @@ def patch_datasets(
     )
 
 
+def __validate_saas_dataset(
```

Comment:
Private methods should only append one `_`

Reply:
Updating to use only one `_`.